src/main/java/reposense/util/FileUtil.java

```diff
@@ -110,6 +111,26 @@ public static void zipFoldersAndFiles(List<Path> pathsToZip,
         }
     }
 
+    /**
+     * Writes the ignore revs file containing the {@code ignoreCommitList} at the given {@code path}.
+     * @return An Optional containing the Path to the ignore revs file, or an empty Optional
+     *         if there was an error while writing the ignore revs file.
+     */
+    public static Optional<Path> writeIgnoreRevsFile(String path, List<CommitHash> ignoreCommitList) {
+        String contentOfIgnoreRevsFile = ignoreCommitList
+                                            .stream()
+                                            .map(CommitHash::toString)
+                                            .reduce("", (hashes, newHash) -> hashes + newHash + "\n");
```

Comment:
The indentation looks weird, I think it should be 8 spaces after the previous line?